### PR TITLE
Changed Test

### PR DESCRIPTION
**Pull request by finn00dev**
test test test test test

### CODE REVIEW

File: test [hunk 1]
```diff
@@ -1 +1,2 @@
-line1edit
\ No newline at end of file
+line1edit
+finn was here
```
Comment (ntorii) [inline test:2]: Test comment. Should have gotten this notification through email.

File: test [hunk 2]
```diff
@@ -1 +1,2 @@
-line1edit
\ No newline at end of file
+line1edit
```
Comment (ntorii) [inline test:1]: Another test comment

**Review by ntorii — COMMENTED** [review]
@finn00dev Make sure you branch off of the develop branch and pull request that new branch, comparing it to the develop branch in this repo.

There's no develop branch in this repo. So don't worry about the above

**Comment by finn00dev** [comment]
I branch off from the develop branch on my forked repo, correct?

**Comment by ktorii** [comment]
Yes. But this specific repo doesn't have a develop branch. Just branch off the master branch for this repo.

**Comment by finn00dev** [comment]
okay thanks ken, git hub is getting pretty familiar now!

**Pull request merged**